code/modules/antagonists/heretic/influences.dm

```diff
@@ -115,7 +115,7 @@
 	icon = 'icons/effects/eldritch.dmi'
 	icon_state = "pierced_illusion"
 	anchored = TRUE
-	interaction_flags_atom = INTERACT_ATOM_NO_FINGERPRINT_ATTACK_HAND
+	interaction_flags_atom = INTERACT_ATOM_NO_FINGERPRINT_INTERACT
```

Comment:
Influences already don't leave behind fingerprints, I've tested it before and i'm very certain
I guess you could make this `INTERACT_ATOM_NO_FINGERPRINT_ATTACK_HAND|INTERACT_ATOM_NO_FINGERPRINT_INTERACT` for the sake of redundancy

Reply:
A player was very insistent on the discord and even indirectly dismissed me by saying 'a heretic that doesn't wipe their rift prints isn't worth collaborating with'. I'm saying that to add context to the dude's confidence in this

Reply:
That person is dumb

Reply:
Double in fact the thing that you changed, `visible_heretic_influence`, NEVER gets interacted with by the heretic, meaning there is no way that it gets fingerprints

Reply:
I think I remember hearing that the actual thing the detective scans is the floors, since sometimes the heretic accidentally clicks on the floor instead of the influence sprite.  Not sure if this is the case, just what I heard.

Reply:
> I think I remember hearing that the actual thing the detective scans is the floors, since sometimes the heretic accidentally clicks on the floor instead of the influence sprite. Not sure if this is the case, just what I heard.

I haven't played det recently, but this IS what we do: it's easy for the heretic to miss the rift sprite and touch the floor

Reply:
> > I think I remember hearing that the actual thing the detective scans is the floors, since sometimes the heretic accidentally clicks on the floor instead of the influence sprite. Not sure if this is the case, just what I heard.
> 
> I haven't played det recently, but this IS what we do: it's easy for the heretic to miss the rift sprite and touch the floor

why does touching the empty floor leave prints

Reply:
Simple fix we just make interacting with the turf the influence is on forward clicks to the influence so it rips your arm off 🙃

Reply:
> > > I think I remember hearing that the actual thing the detective scans is the floors, since sometimes the heretic accidentally clicks on the floor instead of the influence sprite. Not sure if this is the case, just what I heard.
> > 
> > 
> > I haven't played det recently, but this IS what we do: it's easy for the heretic to miss the rift sprite and touch the floor
> 
> why does touching the empty floor leave prints

because you touched it? the floor? with your hands?

Reply:
> Simple fix we just make interacting with the turf the influence is on forward clicks to the influence so it rips your arm off 🙃

We can give influences (both visible and untapped) the component that redirects attack hands on the turf loc to the atom, but I believe fingerprints are applied *before* attack hand comsig.

Reply:
> > > > I think I remember hearing that the actual thing the detective scans is the floors, since sometimes the heretic accidentally clicks on the floor instead of the influence sprite. Not sure if this is the case, just what I heard.
> > > 
> > > 
> > > I haven't played det recently, but this IS what we do: it's easy for the heretic to miss the rift sprite and touch the floor
> > 
> > 
> > why does touching the empty floor leave prints
> 
> because you touched it? the floor? with your hands?

Except you don't do anything on the floor and the game definitely doesn't tell you that you pat the floor like a weirdo and left your fingerprints everywhere.